### ISSUE TITLE
don't run module

### ISSUE THREAD

**Issue by defcoun-myaura** [nico-amsterdam/pop3mail#4]
mix test
===> Fetching rebar3_hex ({pkg,<<"rebar3_hex">>,<<"4.1.0">>})
===> Version cached at /Users/a.kulikov/.cache/rebar3/hex/default/packages/rebar3_hex-4.1.0.tar is up to date, reusing it
===> Compiling rebar3_hex
===> Compiling pop3client
Unchecked dependencies for environment test:
* erlpop (Hex package)
  could not find an app file at "_build/test/lib/erlpop/ebin/erlpop.app". This may happen if the dependency was not yet compiled, or you specified the wrong application name in your deps, or the dependency indeed has no app file (then you can pass app: false as option)
** (Mix) Can't continue due to errors on dependencies

**Comment by nico-amsterdam** [comment]
Bug confirmed:

```elixir
~/elixir/ mix new popmailuser
* creating README.md
* creating .formatter.exs
* creating .gitignore
* creating mix.exs
* creating config
* creating config/config.exs
* creating lib
* creating lib/popmailuser.ex
* creating test
* creating test/test_helper.exs
* creating test/popmailuser_test.exs

Your Mix project was created successfully.
You can use "mix" to compile it, test it, and more:

    cd popmailuser
    mix test

Run "mix help" for more commands.
~/elixir/ cd popmailuser
~/elixir/popmailuser/ vi mix.exs
~/elixir/popmailuser/ mix deps.get
Running dependency resolution...
Dependency resolution completed:
  pop3client 1.3.0
  pop3mail 1.3.1
* Getting pop3mail (Hex package)
  Checking package (https://repo.hex.pm/tarballs/pop3mail-1.3.1.tar)
  Using locally cached package
* Getting erlpop (Hex package)
  Checking package (https://repo.hex.pm/tarballs/pop3client-1.3.0.tar)
  Using locally cached package
~/elixir/popmailuser/ mix test
===> Fetching rebar3_hex ({pkg,<<"rebar3_hex">>,<<"4.1.0">>})
===> Downloaded package, caching at /home/nico/.cache/rebar3/hex/default/packages/rebar3_hex-4.1.0.tar
===> Compiling rebar3_hex
===> Compiling pop3client
==> pop3mail
Compiling 14 files (.ex)
Generated pop3mail app
==> popmailuser
Unchecked dependencies for environment test:
* erlpop (Hex package)
  could not find an app file at "_build/test/lib/erlpop/ebin/erlpop.app". This may happen if the dependency was not yet compiled, or you specified the wrong application name in your deps, or the dependency indeed has no app file (then you can pass app: false as option)
** (Mix) Can't continue due to errors on dependencies
```

**Comment by nico-amsterdam** [comment]
Published pop3mail 1.3.2 with a fix.
Run:
`mix hex.outdated`
to see the latest version. 
Before you upgrade first run:
` mix deps.clean erlpop`
This will remove the old v1.2 erlpop client. I guess that this was the issue, because the hex package was called pop3client, but the app still had the name erlpop. Since erlpop v1.3 the erlang application is also called 'pop3client'. So, no more confusion for hex or mix or us.
After the `mix deps.clean erlpop` command, the deps/erlpop directory should be gone, and also the _build/dev/lib/erlpop and _build/test/lib/erlpop should be vanished.

Then run:
`mix deps.update pop3mail`
to upgrade pop3mail to version 1.3.2 and
verify with:
`   mix hex.outdated`

Prove:
```elixir
~/elixir/ mix new usespopmail                                                
* creating README.md
* creating .formatter.exs
* creating .gitignore
* creating mix.exs
* creating config
* creating config/config.exs
* creating lib
* creating lib/usespopmail.ex
* creating test
* creating test/test_helper.exs
* creating test/usespopmail_test.exs

Your Mix project was created successfully.
You can use "mix" to compile it, test it, and more:

    cd usespopmail
    mix test

Run "mix help" for more commands.
~/elixir/ cd usespopmail 
~/elixir/usespopmail/ sed -i -e 's/# {:dep_from_hexpm.*/{:pop3mail, "~> 1.3.2"}/' mix.exs
~/elixir/usespopmail/ mix deps.get
Running dependency resolution...
Dependency resolution completed:
  pop3client 1.3.1
  pop3mail 1.3.2
* Getting pop3mail (Hex package)
  Checking package (https://repo.hex.pm/tarballs/pop3mail-1.3.2.tar)
  Using locally cached package
* Getting pop3client (Hex package)
  Checking package (https://repo.hex.pm/tarballs/pop3client-1.3.1.tar)
  Using locally cached package
~/elixir/usespopmail/ mix test
===> Compiling pop3client
==> pop3mail
Compiling 14 files (.ex)
Generated pop3mail app
==> usespopmail
Compiling 1 file (.ex)
Generated usespopmail app
..

Finished in 0.1 seconds
1 doctest, 1 test, 0 failures

Randomized with seed 289707

~/elixir/usespopmail/ wget -q https://raw.github.com/nico-amsterdam/pop3mail/master/pop3mail_downloader 
~/elixir/usespopmail/ chmod +x ./pop3mail_downloader 
~/elixir/usespopmail/ ./pop3mail_downloader
mix run -e 'Pop3mail.CLI.main([])'
===> Compiling pop3client
==> pop3mail
Compiling 14 files (.ex)
Generated pop3mail app
==> usespopmail
Compiling 1 file (.ex)
Generated usespopmail app
Please enter your email account name: 
```
